libs/blocks/caas/utils.js

```diff
@@ -208,25 +208,25 @@ const getFilterObj = ({ excludeTags, filterTag, icon, openedOnLoad }, tags, stat
   return filterObj;
 };
 
-const getCustomFilterObj = ({ group, filtersCustomItems, openedOnLoad }) => {
+const getCustomFilterObj = ({ group, filtersCustomItems, openedOnLoad }, strs = {}) => {
   if (!group) return null;
 
   const items = filtersCustomItems.map((item) => ({
     id: item.customFilterTag[0],
-    label: item.filtersCustomLabel,
+    label: item.filtersCustomLabel?.match(/^{.*}$/) ? strs[item.filtersCustomLabel.replace(/{|}/g, '')] : item.filtersCustomLabel || '',
```

Comment:
nit: get line length < 100

Reply:
Since `/^{.*}$/` is used multiple times you can move it out to a const: `const IN_BRACKETS_RE = /^{.*}$/;`

Reply:
Done!

Reply:
@chrischrischris, Can we merge this?